src/browser/app/profile/zen-browser.js

```diff
@@ -162,3 +162,6 @@ pref("sidebar.verticalTabs", false, locked);
 // Better Windows theming
 pref("widget.non-native-theme.scrollbar.style", 2);
 pref("widget.non-native-theme.use-theme-accent", true);
+
+// Closing the last tab should not cause the browser to close, this is bad default inherited from Firefox
+pref("browser.tabs.closeWindowWithLastTab", false);
```

Comment:
It would be best to let the user change this in the settings page imo. Let me know what you think.

Some people like me have gotten heavily used to closing windows by Ctrl+w-ing the last tab.

Reply:
Firefox is the only (major) browser that behave like this by default.

Reply:
@0cool-f We could still have a toggle like this
![image](https://github.com/user-attachments/assets/877c9e7e-969b-4d96-a208-4bf08809729a)

Reply:
I have been using brave for quite some time now and I didn't know that this toggle existed. Sometimes, I have felt that the window should not close after closing the last tab and I never thought that there would be a toggle to switch this. But over time I got used to window closing after last tab, that now it feels slightly weird after using this toggle.